scripts/gen_quick_start_module.py

```diff
@@ -1,23 +1,159 @@
-        return fp.read()
+    with open(BASE_DIR / "_includes" / "quick-start-module.js") as fptr:
+        return fptr.read()
+
+def get_package_type(pkg_key, os_key):
+    package_type = pkg_key
+    if pkg_key == "pip":
+        package_type = "manywheel" if os_key == OperatingSystem.LINUX.value else "wheel"
+    return package_type
 
+def get_gpu_info(acc_key, instr, acc_arch_map):
+    gpu_arch_type, gpu_arch_version = acc_arch_map[acc_key]
+    if DEFAULT in instr:
+        gpu_arch_type, gpu_arch_version = acc_arch_map["accnone"]
+    return (gpu_arch_type, gpu_arch_version)
 
+# This method is used for generating new published_versions.json file
+# It will modify versions json object with installation instructions
+# Provided by generate install matrix Github Workflow, stored in release_matrix
+# json object.
+def update_versions(versions, release_matrix, release_version):
+    version = "preview"
+    acc_arch_map = acc_arch_ver_map[release_version]
+
+    if release_version != "nightly":
+        version = release_matrix[OperatingSystem.LINUX.value][0]["stable_version"]
+        if version not in versions["versions"]:
+            versions["versions"][version] = copy.deepcopy(versions["versions"]["preview"])
+            versions["latest_stable"] = version
+
+    # Perform update of the json file from release matrix
+    for os_key, os_vers in versions["versions"][version].items():
+        for pkg_key, pkg_vers in os_vers.items():
+            for acc_key, instr in pkg_vers.items():
+                package_type = get_package_type(pkg_key, os_key)
+                gpu_arch_type, gpu_arch_version = get_gpu_info(acc_key, instr, acc_arch_map)
+
+                pkg_arch_matrix = [
+                    x for x in release_matrix[os_key]
+                    if (x["package_type"], x["gpu_arch_type"], x["gpu_arch_version"]) ==
+                    (package_type, gpu_arch_type, gpu_arch_version)
+                    ] or None
+
+                if pkg_arch_matrix is not None:
+                    if package_type != "libtorch":
+                        instr["command"] = pkg_arch_matrix[0]["installation"]
+                    else:
+                        if os_key == OperatingSystem.LINUX.value:
+                            rel_entry_pre_cxx1 = next(
+                                x for x in pkg_arch_matrix
+                                if x["devtoolset"] == PRE_CXX11_ABI
+                             )
+                            rel_entry_cxx1_abi = next(
+                                x for x in pkg_arch_matrix
+                                if x["devtoolset"] == CXX11_ABI
+                                )
+                            if instr["versions"] is not None:
+                                instr["versions"][LIBTORCH_DWNL_INSTR[PRE_CXX11_ABI]] = rel_entry_pre_cxx1["installation"]
+                                instr["versions"][LIBTORCH_DWNL_INSTR[CXX11_ABI]] = rel_entry_cxx1_abi["installation"]
```

Comment:
Isn't the below more concise and avoids duplications?
```suggestion
                            rel_entry_dict = {x["devtoolset"]: x["installation"] for x in pkg_arch_matrix}
                            if instr["versions"] is not None:
                                for ver in [PRE_CXX11_ABI, CXX11_ABI]:
                                    instr["versions"][LIBTORCH_DWNL_INSTR[ver]] = rel_entry_dict[ver]
```